src/types/main.d.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * For default interfaces that are frequently used and extensions:
+ *   - HTMLElement: h5s and its inner objects.
+ *   - Extension of objects that are external to the library.
+*/
+
+interface opts {
+    connectWith: boolean,
+    acceptFrom: void,
+    copy: boolean,
+    placeholder: void,
+    disableIEFix: boolean,
+    placeholderClass: string,
+    draggingClass: string,
+    hoverClass: boolean,
+    debounce: number,
+    maxItems: number,
+    itemSerializer: void,
+    containerSerializer: void,
+    items: string
+}
+
+// Can these functions be better defined?
+interface events {
```

Comment:
I think this is not needed, as we can use `DragEvent` which should have those functions, right?

Reply:
ok deleted